proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
+
+== CN4J
+Pros
+
+* Allows for parties interested in both MPWG and CN4J/Jakarta to sign one working group participation agreement
+* Addresses IP gap in existing MP development
+* Tends to focus on stability and backwards compatibility 
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+
+Cons
+
+* Currently has a higher entry cost and overall overhead to run
+* Currently has no zero entry participation model for larger companies to sponsor specification development
+* Introduces a conflict of interest between evolution and support of enterprise customers and their previous investment in Java EE with a shared steering committee
```

Comment:
I don't know that the use of a common WG introduces this conflict of interest.  Many of us have been participating in both camps for the last few years without a conflict of interest.  I would at least soften this with "potential".

Reply:
I don;t believe this is correct.

Reply:
I think we are trying to image conflict of interests. To me, sharing one steering committee facilities better alignment.

Reply:
MP does not have a steering committee today. The Jakarta steering committee is very much focused on Jakarta EE for obvious reasons. If there was to be a single WG for both efforts the a new Steering Committee would have to be formed which has a different remit to the Jakarta EE one.

Reply:
Of course it would. That entire charter for what is currently known as the Jakarta EE working group would have to be overhauled in this scenario. That is already explicit in the proposal.

The entire suggestion that MP's spec process and release cadence would be different in the CN4J proposal versus the MPWG proposal is also a red herring. Under CN4J, MP would have complete control over its own spec process and release cadence. To imply otherwise is just FUD.

Reply:
Both uncertainty and doubt exist due to at least two key points of conflict between the proposals not being defined; how a shared steering committee defines membership and votes, and what the membership fee structure is. These need to be addressed before a meaningful comparison can be done.